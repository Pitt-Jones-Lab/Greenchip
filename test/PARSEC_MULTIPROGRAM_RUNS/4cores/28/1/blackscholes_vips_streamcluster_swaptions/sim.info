{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/blackscholes_vips_streamcluster_swaptions',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_1_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/blackscholes_vips_streamcluster_swaptions',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpJjTWYd/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb19',
 'pin_version': 'unknown',
 'rusage': (16616.760868000001,
            611.96396700000003,
            198552,
            0,
            0,
            0,
            47269,
            1,
            0,
            61480,
            1216,
            0,
            0,
            0,
            65054562,
            5677953),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/blackscholes_vips_streamcluster_swaptions --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem28.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_1_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpJjTWYd/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/blackscholes_vips_streamcluster_swaptions/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 7225.1019189357758,
 't_start': 1463065427.4356041,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1630060544L}
