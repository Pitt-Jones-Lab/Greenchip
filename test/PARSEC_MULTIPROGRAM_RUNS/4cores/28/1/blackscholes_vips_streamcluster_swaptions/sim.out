                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2392513562 |  2478320191 |  2671579211 |  2456247536
  Cycles                           |  1446841830 |  1446841830 |  1446841830 |  1446841830
  IPC                              |        1.65 |        1.71 |        1.85 |        1.70
  Time (ns)                        |   543925500 |   543925500 |   543925500 |   543925500
  Idle time (ns)                   |       76696 |      571551 |       57256 |       83487
  Idle time (%)                    |        0.0% |        0.1% |        0.0% |        0.0%
Branch predictor stats             |             |             |             |            
  num correct                      |   242636424 |   238259197 |   193250524 |   237240520
  num incorrect                    |     6281471 |     4313236 |     2875467 |     4542300
  misprediction rate               |       2.52% |       1.78% |       1.47% |       1.88%
  mpki                             |        2.63 |        1.74 |        1.08 |        1.85
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   324535440 |   329509118 |   383462285 |   326243657
    num misses                     |        3430 |        2028 |        1286 |        1412
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   788557298 |   808156605 |  1001007916 |   804651830
    num misses                     |      155451 |     1004259 |      131762 |      154589
    miss rate                      |       0.02% |       0.12% |       0.01% |       0.02%
    mpki                           |        0.06 |        0.41 |        0.05 |        0.06
  L2 TLB                           |             |             |             |            
    num accesses                   |      158881 |     1006287 |      133048 |      156001
    num misses                     |       48654 |       77959 |       60866 |       53468
    miss rate                      |      30.62% |       7.75% |      45.75% |      34.27%
    mpki                           |        0.02 |        0.03 |        0.02 |        0.02
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   324535440 |   329509118 |   383462285 |   326243657
    num cache misses               |      245341 |      221678 |      146500 |      198865
    miss rate                      |       0.08% |       0.07% |       0.04% |       0.06%
    mpki                           |        0.10 |        0.09 |        0.05 |        0.08
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   788557298 |   808156605 |  1001007916 |   804651830
    num cache misses               |     8215749 |     9616782 |     7069952 |     8990446
    miss rate                      |       1.04% |       1.19% |       0.71% |       1.12%
    mpki                           |        3.43 |        3.88 |        2.65 |        3.66
  Cache L2                         |             |             |             |            
    num cache accesses             |     9331358 |    10865164 |     8892744 |    10185940
    num cache misses               |     2280275 |     2835182 |     2956971 |     2548254
    miss rate                      |      24.44% |      26.09% |      33.25% |      25.02%
    mpki                           |        0.95 |        1.14 |        1.11 |        1.04
DRAM summary                       |             |             |             |            
  num dram accesses                |    19799546 |           0 |           0 |           0
  average dram access latency (ns) |       87.43 |         inf |         inf |         inf
  average dram queueing delay      |       87.43 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     1929816 |     2286896 |     2493293 |     2229041
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
